.rubocop_local.yml

```diff
@@ -5,6 +5,7 @@ GlobalVars:
   - $api_log
   - $aws_log
   - $azure_log
+  - $dw_log
```

Comment:
@durandom @moolitayer actually this seems specific (not a type), so I expect it to be "hawkular_log" probably.